tls/s2n_alerts.c

```diff
@@ -279,8 +286,17 @@ S2N_RESULT s2n_alerts_write_error_or_close_notify(struct s2n_connection *conn)
         return S2N_RESULT_OK;
     }
 
-    /* By default, s2n-tls sends a generic close_notify alert, even in
-     * response to fatal errors.
+    /*
+     *= https://tools.ietf.org/rfc/rfc8446#section-6.2
+     *= type=exception
+     *# The phrases "terminate the connection with an X
```

Comment:
Add a "reason" field. It can just contain an abbreviated version of the full explanation, but it'll help if our goal is to document this via duvet.

Reply:
yep just saw reason in some other exceptions also